core/src/dunit/scala/io/snappydata/cluster/SplitClusterDUnitTest.scala

```diff
@@ -688,31 +689,20 @@ object SplitClusterDUnitTest extends SplitClusterDUnitTestObject {
     val hostName = InetAddress.getLocalHost.getHostName
     val sparkShellCommand = s"$sparkProductDir/bin/spark-shell --master spark://$hostName:7077" +
         " --conf spark.snappydata.connection=localhost:" + locatorClientPort +
-        s" --jars $snappyDataCoreJar" +
-        securityConf +
-        s" -i $scriptFile -i $scriptFile2"
-
-    logInfo(s"Invoking spark-shell: $sparkShellCommand")
-    val conn = getConnection(locatorClientPort, props)
-    val stmt = conn.createStatement()
-
-    runSparkShellSnappyPoolTest(stmt, sparkShellCommand)
-
-    // accessing tables created through spark-shell
-    val rs1 = stmt.executeQuery("select count(*) from coltable")
-    rs1.next()
-    assert(rs1.getInt(1) == 5)
+        s" --jars $snappyDataCoreJar" + securityConf
 
-    val rs2 = stmt.executeQuery("select count(*) from rowtable")
-    rs2.next()
-    assert(rs2.getInt(1) == 5)
+    // run SparkSqlTestCode.txt command file on spark-shell
+    val sparkSqlScriptCommand = sparkShellCommand + s" -i $scriptFile"
+    runSparkShellSnappyPoolTest(sparkSqlScriptCommand)
+    // access table created via spark-shell
+    accessTableCreatedViaSparkShell(locatorClientPort, props, "coltable", "rowtable" , 5)
 
-    // drop the tables
-    stmt.execute("drop table rowtable")
-    stmt.execute("drop table coltable")
+    // run SnappySqlPoolTestCode.txt command file on spark-shell
+    val snappySqlScriptCommand = sparkShellCommand + s" -i $scriptFile2"
+    runSparkShellSnappyPoolTest(snappySqlScriptCommand)
+    // access table created via spark-shell
+    accessTableCreatedViaSparkShell(locatorClientPort, props)
```

Comment:
Why two separate runs of spark-shell? Two scripts on same command-line works just fine. Will unnecessarily increase running time of the test.